dohq_artifactory/admin.py

```diff
@@ -271,6 +272,11 @@ def encryptedPassword(self):
         Method for backwards compatibility, see property encrypted_password
         :return:
         """
+        # todo remove around April 2022
+        warn(
```

Comment:
Can we create `deprecation` function like this? https://docs.python.org/3/library/warnings.html#warnings.warn

```
def deprecation(message):
    warnings.warn(message, DeprecationWarning, stacklevel=2)
```

Reply:
done